src/api/todos.ts

```diff
@@ -0,0 +1,15 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
```

Comment:
As this function has only one statement, if you want, we could return directly without using the `return` keyword. Like this: 
```js
export const getTodos = (userId: number) => client.get<Todo[]>(`/todos?userId=${userId}`);
```